### ISSUE TITLE
Global Shields preferences not respected on some sites

### ISSUE THREAD

**Issue by GuardianMajor** [brave/brave-browser#7163]
## Description 

There is no obvious way to disable the shields globally, the reason why is irrelevant since we are giving the users choice right? Just that it is not doing it. Best I can see this should disable it unless the option is buried elsewhere and there really should be an option that is unambiguous and just works:

![image](https://user-images.githubusercontent.com/5274181/69921018-4159f400-1443-11ea-85dc-6d0c4fc0598e.png)

## Steps to Reproduce

   1. Configure as above
   2. Go to any site and the global shield is in effect
   3. Simple really

## Actual result:

As you can see, the shield is globally enabled on every site, despite the option above clearly showing that it shouldn't be enforcing `Cross-site trackers` blocking.

![image](https://user-images.githubusercontent.com/5274181/69921086-060bf500-1444-11ea-9602-40a5a522e8d1.png)

## Expected result:
Simple, you disable it, it is disabled, no ifs ands and buts right? It is not. Unless I am doing something incorrectly which clearly based on the option showed above, I am not - so why is it outright ignoring it?

## Reproduces how often: 

Every single time, every single site - no exceptions (unless you manually do each site which comes back and enables the next time automatically, much like this comment: https://github.com/brave/brave-browser/issues/2142#issuecomment-438808951 so pretty much moot and very unnecessary burden to configure each and every site anyway)

## Brave version (brave://version info)

![image](https://user-images.githubusercontent.com/5274181/69921182-c1cd2480-1444-11ea-8d68-cc9999a9b1c0.png)

## Version/Channel Information:

I have honestly not tried because I don't want to run a bunch of different versions when the release should just work.

- Can you reproduce this issue with the current release? **YES**
- Can you reproduce this issue with the beta channel? **DON'T KNOW**
- Can you reproduce this issue with the dev channel? **DON'T KNOW**
- Can you reproduce this issue with the nightly channel? **DON'T KNOW**

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? **The problem IS that you cannot disable the shields, so a bit redundant**
- Does the issue resolve itself when disabling Brave Rewards? **Apparently not since it is already disabled**
- Is the issue reproducible on the latest version of Chrome? **Since Chrome doesn't have this feature so far as I know and this is supposed to be better than Google, I am going on a limb to say no**

## Miscellaneous Information:

Don't know what else to provide that would help you but if you need something, let me know and I will happily provide it.

Thanks in advance.

**Comment by franverona** [comment]
For the record and since this morning, some sites are showing me ads (including Youtube, which it wasn't displaying any of them at all yesterday). Idk if this issue is related with my problem, or if anyone is also having it. I tried in different devices with same result.

**Comment by bsclifton** [comment]
cc: @rebron @tomlowenthal

**Comment by franverona** [comment]
Small update: today Brave is working as expected and all ads (including Youtube) are being blocked successfully.

**Comment by tomlowenthal** [comment]
Are you still seeing the issue where your global Shields defaults aren't being respected on new sites?

**Comment by GuardianMajor** [comment]
Yes, I tried a site I have not gone to in a while and certainly not on this browser and it still shows up globally on it:

![image](https://user-images.githubusercontent.com/5274181/70016080-33d06700-1534-11ea-91c7-71780c4291ad.png)

I can assure that it is not site-specific either but wouldn't know where to check those anyway, there is no interface that I can see.

**Comment by Clubspeed-Mike** [comment]
@tomlowenthal That change in title is incorrect and misleading as it is not "some sites" it is ALL SITES, the original title was more accurate and to the point. And it is not simply about respecting the shield settings but rather the abject inability to just disable it period and or even review the sites that were manually allowed, I don't see any place for that either.

**Comment by GuardianMajor** [comment]
@tomlowenthal The title is incorrect and misleading as it is not "some sites" it is ALL SITES, the original title was more accurate and to the point. And it is not simply about respecting the shield settings but rather the abject inability to just disable it period and or even review the sites that were manually allowed, I don't see any place for that either.

**Comment by bsclifton** [comment]
The root cause for this issue is likely https://github.com/brave/brave-browser/issues/5721

Basically, once an individual site has shields toggled, it stores whatever the last setting was.
To solve this problem, we should update the browser to delete the site setting if it matches the default - letting the global one take precedence.

Closing this issue as a duplicate of https://github.com/brave/brave-browser/issues/5721
